### PR TITLE
Resolve VCSWP-20372 (TranscribeUtterance not generating as a PerclCommand)

### PR DESCRIPTION
**Pull request by amahadaya**
Closes VCSWP-20372

### CODE REVIEW

File: openapi.json
Comment (ngarg-vail) [inline openapi.json:3337]: Why is this appearing as a diff in this way? shouldn't there just be an `x-percl-command` update? Also it shouldn't be removing properties unless that was an intended change.

File: test/test_transcribe_utterance.py
Comment (ngarg-vail) [inline test/test_transcribe_utterance.py:48]: Why is this removed?

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: freeclimb/model/transcribe_utterance.py
Comment (ngarg-vail) [inline freeclimb/model/transcribe_utterance.py]: Returning twice. this is wrong.

File: openapi.json
Comment (amahadaya) [reply openapi.json:3337]: This is an intended change, Logan had removed the x-max-lengths properties and this openapi.json file did not have this change, so once autogenerated, it applied the openapi changes from the latest branch to this file

File: test/test_transcribe_utterance.py
Comment (amahadaya) [reply test/test_transcribe_utterance.py:48]: I believe latest commit should add the test back in

File: openapi.json
Comment (ngarg-vail) [inline openapi.json]: we need to remove this from the spec *asap*

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

**Review by ngarg-vail — APPROVED** [review]

File: freeclimb/model/transcribe_utterance.py
Comment (ngarg-vail) [reply freeclimb/model/transcribe_utterance.py]: Needs to be resolved before approval

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (ngarg-vail) [inline CHANGELOG.md]: remove

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

**Review by ngarg-vail — APPROVED** [review]

**Pull request merged**
